src/protocol/profiler.rs

```diff
@@ -0,0 +1,87 @@
+use serde::Deserialize;
+
+#[derive(Debug, Clone, Deserialize)]
+#[serde(rename_all = "camelCase")]
+/// Coverage data for a source range.
+pub struct CoverageRange {
+    pub start_offset: u16,
```

Comment:
The original docs for these items are

`JavaScript script source offset for the range start.`
`JavaScript script source offset for the range end.`
`Collected execution count of the source range.`

Which are always helpful